igdbHelper.js

```diff
@@ -0,0 +1,70 @@
+require("dotenv").config();
+const axios = require("axios");
+
+const { CLIENT_ID, CLIENT_SECRET } = process.env;
+
+let tokenExperationTime;
+let accessToken;
+
+//sets token and timer, if token doesn't exist and/or is expired.
+const getAccessToken = async () => {
+  const currentTime = new Date().getTime();
+  if (accessToken && currentTime < tokenExperationTime) return;
+
+  const url = `https://id.twitch.tv/oauth2/token?client_id=${CLIENT_ID}&client_secret=${CLIENT_SECRET}&grant_type=client_credentials`;
+  const response = await axios.post(url);
+
+  accessToken = response.data.access_token;
+  tokenExperationTime = response.data.expires_in + currentTime;
+};
+
+const reusableAxiosPost = async (routeName) => {
```

Comment:
I think we can go with a more generic name here. Maybe just `IDGB_Post`, or just `Post`.